gradle.properties

```diff
@@ -1,2 +1,2 @@
 group=org.ballerinalang
-version=0.8.11-SNAPSHOT
+version=0.8.15-SNAPSHOT
```

Comment:
Since we released 0.8.11 tool version along with the SL Alpha, shouldn't this be `0.8.12-SNAPSHOT`?

Reply:
Latest tool version is 0.8.14
https://github.com/ballerina-platform/ballerina-update-tool/packages/136624